[Docs] update How To 1 (render plane manipulator)

psobolewskiPhD opened the pull request:
Closes: https://github.com/napari-threedee/napari-threedee/issues/205

In the PR I correct the napari.yaml display name of the sample to be consistent and do minor copy-edits to the text:
1. provide a link to Zenodo
2. warn about sample file size
3. warn that there may be subtle visual differences depending on versions of n3d and napari
4. specify that double-click moves the manipulator (that is the actual napari shortcut, shift-click working is technically a bug? see https://github.com/napari/napari/blob/main/napari/layers/image/_image_mouse_bindings.py )

GenevieveBuckley commented on docs/how_to/render_plane_manipulator.md:
````diff
@@ -19,7 +19,7 @@ the rendering plane.
      ```
 
-2. Load the "HIV virus-like particles tomogram" image from the File menu: File -> Open Sample -> HIV virus-like particle tomogram. Note that this will download the sample image so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
+2. Load the "HIV virus-like particles tomogram" image from the File menu: **File -> Open Sample -> HIV virus-like particles tomogram**. Note that this will download the [sample image from Zenodo](https://doi.org/10.5281/zenodo.6504891) (~470 Mb) so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
````
This part of the documentation is fragile, because as soon as there is more than one example dataset provided, then they'll be found in a subfolder, eg: File -> Open Sample -> napari-threedee -> HIV virus-like particle tomograms
I don't know what my suggestion is here, either we ignore this possibility for now, or force the change to happen sooner by adding a second sample dataset.

kevinyamauchi replied:
Ah good point. I think we can leave as is and then update if/when we add additional sample data plugins

GenevieveBuckley commented on docs/how_to/render_plane_manipulator.md:
````diff
@@ -19,7 +19,7 @@ the rendering plane.
      ```
 
-2. Load the "HIV virus-like particles tomogram" image from the File menu: File -> Open Sample -> HIV virus-like particle tomogram. Note that this will download the sample image so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
+2. Load the "HIV virus-like particles tomogram" image from the File menu: **File -> Open Sample -> HIV virus-like particles tomogram**. Note that this will download the [sample image from Zenodo](https://doi.org/10.5281/zenodo.6504891) (~470 Mb) so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
````
```suggestion
2. Load the "HIV virus-like particles tomogram" image from the File menu: **File -> Open Sample -> HIV virus-like particle tomogram**. Note that this will download the [sample image from Zenodo](https://doi.org/10.5281/zenodo.6504891) (~470 Mb) so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
```

Typo, it should be "particle" not "particles", to match what is actually shown in the menu

GenevieveBuckley replied:
Extra suggestion: I think people will avoid clicking the zenodo link because the text here implies it will start an automatic download. Can we re-phrase to make it more obvious the link is to the data description page instead?

Eg:
...this will download the sample image from Zenodo (~470 Mb), for more details see the [data description page here](https://doi.org/10.5281/zenodo.6504891).

GenevieveBuckley commented on docs/how_to/render_plane_manipulator.md:
```diff
@@ -39,7 +39,7 @@ the rendering plane.
 	![plane rendering mode](https://user-images.githubusercontent.com/1120672/207311620-0494f9cf-3059-40d4-902e-37f026556c56.png)
 
-5. Open the render plane manipulator plugin from the menu: Plugins -> napari-threedee -> render plane manipulator. You will see the render plane manipulator appear on the rendering plane.
+5. Open the render plane manipulator plugin from the menu: **Plugins -> napari-threedee -> render plane manipulator**. You will see the render plane manipulator appear in the corner of the rendering plane. You can double-click with the left mouse button (or hold down the Shift key while clicking) to reposition the manipulator. Note that depending on your versions of `napari-threedee` and napari t ...
```
Don't you need to click the "activate" button first, before you can see the render plane manipulator? That seems like an important point to mention, in all of these tutorials

kevinyamauchi replied:
Good point!

psobolewskiPhD commented on docs/how_to/render_plane_manipulator.md:
````diff
@@ -19,7 +19,7 @@ the rendering plane.
      ```
 
-2. Load the "HIV virus-like particles tomogram" image from the File menu: File -> Open Sample -> HIV virus-like particle tomogram. Note that this will download the sample image so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
+2. Load the "HIV virus-like particles tomogram" image from the File menu: **File -> Open Sample -> HIV virus-like particles tomogram**. Note that this will download the [sample image from Zenodo](https://doi.org/10.5281/zenodo.6504891) (~470 Mb) so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
````
There arn't really any more details there. 
I'll put the DOI link there instead of having it be hidden. I don't think anyone would expect clicking a DOI to download anything, it's a record locator after all.

psobolewskiPhD commented:
I addressed the comments as follows:
- For the Zenodo link, I just provide the DOI as a clickable link, rather than obscuring and move the 470 Mb reference be by the sample image and not the link.
- For the "activate" point, I clarified the opening of the widget and clicking activate.
